fvm/src/kernel/default.rs

```diff
@@ -539,6 +541,22 @@ where
             .call_manager
             .charge_gas(self.call_manager.price_list().on_verify_post(verify_info))?;
 
+        // Due to a bug on calibrationnet, we have some _valid_ StackedDRGWindow32GiBV1P1
+        // proofs that were deemed invalid on chain. This was fixed WITHOUT a network version check.
+        // As a result, we need to explicitly consider all such proofs invalid, ONLY on calibrationnet,
+        // and ONLY before epoch 498691,
+        let calibnet_chain_id = ChainID::from(314159);
+
+        #[allow(clippy::collapsible_if)]
+        if self.call_manager.context().network.chain_id == calibnet_chain_id {
```

Comment:
Any reason not to collapse these checks? `rustfmt` should align everything nicely. Also, we can get a reference to the context first:

```rust
let ctx = self.call_manager.context();
if ctx.network.chain_id == calibnet_chain_id
    && ctx.epoch <= 498691
    && ... {
```

Reply:
@Stebalien I wanted to explicitly have it wrapped in a "ARE YOU CALIBNET" top-level block. But we are biking all over this shed.

Agreed with getting context first.

Reply:
I don't _really_ care. I just mildly prefer appeasing the linter rather than fighting it.

Reply:
But you're right, it makes sense to have a top-level check.